specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/networkManagerActiveConfiguration.json

```diff
@@ -0,0 +1,221 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagerActiveConfiguration",
+    "description": "The Microsoft Azure Network Manager Activity Configuration API provides a RESTful set of web services that interact with Microsoft Azure Network Manager service to manage your network resources.",
+    "version": "2020-08-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/activeConfigurations/{locationName}": {
```

Comment:
Where is the PUT api for activeConfigurations?

Reply:
If activeConfigurations is not a nested resource, please expose this API as a post method. Consider passing locationName in the filter query parameter.

Reply:
activeConfigurations is not a nested operation. It is used to list all configurations. Still, any reason we should use POST instead of get here?

Reply:
Had an internal conversation, using GET is the recommended pattern. Using locationName in the URI implies a GET on a single object. But as this GET response is returning multiple resources, the locationName should be used as query filter parameter.